test-configs.yaml

```
@@ -183,27 +183,47 @@ test_plans:
     params:
       job_timeout: '120'
 
-  ltp-crypto:
+  ltp-opts: &ltp-rootfs
```

```suggestion
  ltp: &ltp
```
This could be used as the "base" test plan definition, which is not a rootfs definition.